python-avd/pyavd/_eos_designs/structured_config/network_services/router_bgp.py

```diff
@@ -287,6 +288,18 @@ def _update_router_bgp_vrf_evpn_or_mpls_cfg(
         vrf_rt = self.get_vrf_rt(vrf)
         route_targets = {"import": [], "export": []}
 
+        # if "evpn" in vrf_address_families the RTs will be added
```

Comment:
lets append "evpn" to the input list of `vrf_ddres_families` instead (if not already there) -> maybe change it to set actually